tracer/src/Datadog.Trace/Telemetry/Collectors/ConfigurationTelemetryCollector.cs

```diff
@@ -121,13 +122,14 @@ public ICollection<TelemetryValue> GetConfigurationData()
 
             var settings = _settings.Settings;
 
-            var data = new List<TelemetryValue>(settings.IsRunningInAzureAppService ? 31 : 26)
+            var data = new List<TelemetryValue>(27 + (settings.IsRunningInAzureAppService ? 5 : 0))
```

Comment:
Made this change so we only need to update one number when we add a new config value